packages/rum-core/test/performance-monitoring/capture-navigation.spec.js

```diff
@@ -194,28 +208,101 @@ describe('Capture hard navigation', function() {
         _start: 0,
         _end: 2119.7900000006484,
         ended: true
+      }),
+      jasmine.objectContaining({
+        name: 'measure_5',
+        type: 'app',
+        _start: 0,
+        _end: 100.7900000006484,
+        ended: true
       })
     ])
   })
 
-  it('should captureHardNavigation', function() {
-    var tr = new Transaction('test', 'test')
-    tr.isHardNavigation = true
+  it('should capture spans for hard navigation', function() {
+    const tr = new Transaction('test', PAGE_LOAD)
```

Comment:
for hard navigation we still need both `PAGE_LOAD` and `captureTimings` since we do not want to capture navigation timings as part of custom/route-change transaction.